dotCMS/src/functional-test/java/com/dotmarketing/portlets/rules/RulesUnderPageAssetsFTest.java

```diff
@@ -185,11 +187,13 @@ public void copyPageWithRules() throws Exception {
         //Remove Page with rules.
         APILocator.getContentletAPI().unpublish(dummyPage, sysUser, false);
         APILocator.getContentletAPI().archive(dummyPage, sysUser, false);
+        Thread.sleep(TimeUnit.SECONDS.toMillis(ALLOWED_SECONDS_FOR_INDEX_SYNC));
```

Comment:
There's a method called contentletAPI.isInodeIndexed that is usually used for this purpose